rlclientlib/factory_resolver.cc

```diff
@@ -170,6 +197,13 @@ void factory_initializer::register_default_factories()
         const char* file_name = c.get(name::INTERACTION_FILE_NAME, "interaction.fb.data");
         return file_sender_create(retval, c, file_name, cb, trace_logger, status);
       });
+
+  // Register a default factory for LOCAL_LOOP_SENDER that returns an error
```

Comment:
does this also need to go behind an `ifdef`?

Reply:
This is just to print out a more helpful error message when the option is used incorrectly, so I'm putting it outside of the ifdef. That way, you'll still get the error message if you load a config into a rlclientlib compiled without FL.